ext_emconf.php

```diff
@@ -28,12 +28,12 @@
     'category'         => 'plugin',
     'author'           => 'effective WEBWORK GmbH',
     'author_email'     => 'info@effective-webwork.de',
-    'state'            => 'stable',
+    'state'            => 'development',
```

Comment:
There is no state "development". Have a look at https://docs.typo3.org/typo3cms/CoreApiReference/6.2/ExtensionArchitecture/DeclarationFile/

It's for sure only "beta", but you shouldn't touch this state with your patch and diskuss this in another issue.